src/metabase/api/card.clj

```diff
@@ -784,7 +784,8 @@ saved later when it is ready."
    :context      (if collection_preview :collection :question)
    :middleware   {:process-viz-settings? false}))
 
-(api/defendpoint ^:streaming POST "/:card-id/query/:export-format"
+(api/defendpoint ^:streaming ^{:content-types #{:content/json :content/form}} POST
```

Comment:
Is there ever going to be something in this set that doesn't start with `:content/`? Seems like noise

Reply:
My plan is that they would all be `:content/<something>`. I think this should make it easier to find all usages of the keywords. If we revert to un-namespaced, finding where this stuff works by looking for `:json` would be quite difficult. I hope this doesn't appear as noise because i consider it signal. If not, we could move to longer un-namespaced keywords but they would probably be similar to `:content-json` at which point i think we aren't gaining anything.